quupe-RN/js/screens/Borrow/BorrowContainer.js

```diff
@@ -1,14 +1,36 @@
 import React, { Component } from 'react';
-import { View, Text } from 'react-native';
+import { View } from 'react-native';
 
 import Borrow from './Borrow';
 
+// Maps is thanks to https://medium.com/@princessjanf/react-native-maps-with-direction-from-current-location-ab1a371732c2
 class BorrowContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
we are going to move this (state info) to either Redux or to the server soon (not with this pull request).